test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,33 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // objects with no null
+        assertIsNotNull(1);
+        assertIsNotNull("abc");
+        assertIsNotNull(Integer.valueOf(1));
```

Comment:
Can test for different number of arguments and combinations of data types as well.
eg. no arguments, (String, int, int), (String,String) etc.